Resources/Prototypes/Catalog/Fills/Items/briefcases.yml

```diff
@@ -38,5 +38,13 @@
   - type: StorageFill
     contents:
       - id: ClothingEyesGlassesSunglasses
-      - id: SpaceCash1000
-        amount: 5
+      - id: SpaceCash5000
+        amount: 6
```

Comment:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/074f776e-f54d-4d66-b9d0-fc14d2929355)

Also, you can't make more items than you can fit in the suitcase. I understand that they are automatically stacked in one slot, but it would be better if you make a new prototype of the SpaceCash30000 item.